db/migrate/20230221015706_add_comments_counter_to_post.rb

```diff
@@ -0,0 +1,6 @@
+class AddCountersToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_column :posts, :comments_counter, :integer, default: 0
+    add_column :posts, :likes_counter, :integer, default: 0
+  end
+end
```

Comment:
- [ ] Good job making the requested changes by the previous reviewer, however this has introduced errors when migrating the database, kindly fix this
![image](https://user-images.githubusercontent.com/81932478/220231712-5565bf00-7a56-455f-94fa-a4c644276b68.png)